modules/installation/pages/hardware-requirements.adoc

```diff
@@ -22,6 +22,7 @@ For {smr} hardware requirements, see xref:retail:retail-requirements.adoc[].
 | Disk Space:              | [path]``/`` _(root)_ Minimum 24{nbsp}GB
 |                          | [path]``/var/lib/pgsql`` Minimum 50{nbsp}GB
 |                          | [path]``/var/spacewalk`` Minimum 50{nbsp}GB per SUSE product and 360{nbsp}GB per Red Hat product
+| Swap space:               | 2-3{nbsp}GB
```

Comment:
How about just settling on a value, let's say 3 to stay abundant?

Reply:
Changed it to 3GB.